### ISSUE TITLE
run scss-lint from node with glob patterns

### ISSUE THREAD

**Issue by juanfran** [brigade/scss-lint#596]
I'm trying to run scss-lint from node but when I use glob patterns, It doesn't work.

```
scss-lint test/**/*.scss
```

If I run this command works well but If I run the same command from node in the same directory I get `No such file or directory - test/**/*.scss`

```js
var child_process = require('child_process');

var command = 'scss-lint test/**/*.scss';

child_process.exec(command, {}, function(error, report) {
  console.log(report);
});
```

Even if I add the full path It doesn't work. But with the filename without glob it works.

Thanks!!

**Comment by sds** [comment]
Hey @juanfran,

Glob patterns are expanded by the shell, which according to the [Child Process documentation](https://nodejs.org/api/child_process.html#child_process_child_process_exec_command_options_callback) defaults to `/bin/sh` on UNIX systems. Depending on the distribution you're running, this could be `bash`, `dash`, or some other shell&mdash;in your case it's one which doesn't seem to support glob expansion.

Remember that files can technically have asterisks (`*`) in their name, so `scss-lint` can't perform this expansion for you. Please figure out on your side which shell you should be using to get this to work.

Hope that helps.
